.python-version

```diff
@@ -0,0 +1 @@
+3.5.7
```

Comment:
I assume this file was checked in by mistake? Since we don't require a specific python version.

Reply:
Thanks, yes. Let me remove that.